### PR TITLE
Seed locales with places

### PR DESCRIPTION
**Pull request by ericcecchi**
- Adds service to get Places from the Places API
- Adds interaction to create a Place
- Adds interaction to create a Locale
- Seeds `locales` and `places` with some data

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,10 +3,12 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
+gem 'active_interaction', '~> 4.0'
```
Comment (ericcecchi) [inline Gemfile:6]: This is a gem that helps with abstracting business logic and making it reusable.

File: Gemfile [hunk 2]
```diff
@@ -3,10 +3,12 @@
...
 gem 'after_party'
 gem 'bcrypt',        '3.1.13'
 gem 'bootsnap',      '1.7.2', require: false
 gem 'faker',         '2.11.0'
+gem 'google_places', '2.0.0'
```
Comment (ericcecchi) [inline Gemfile:11]: This gem is a wrapper for the Places API.

File: Gemfile [hunk 3]
```diff
@@ -26,6 +29,7 @@ group :development do
   gem 'brakeman'
   gem 'bundler-audit'
   gem 'listen', '3.4.1'
+  gem 'pry-byebug'
```
Comment (ericcecchi) [inline Gemfile:32]: This is a helpful debugging tool.

File: app/interactions/locales/create.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
+module Locales
+  # Interaction to create a locale
+  class Create < ApplicationInteraction
+    hash :params do
+      string :name
+      string :content
+      string :hero_image_url
+    end
+    string :external_place_id, default: nil
```
Comment (ericcecchi) [inline app/interactions/locales/create.rb:9]: Interactions specify what parameters they take and the validate their types.

File: app/interactions/locales/create.rb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+    attr_reader :place, :locale
+
+    def execute
```
Comment (ericcecchi) [inline app/interactions/locales/create.rb:13]: All interactions have an `execute` method that is called when they are `.run` or used in `compose`.

File: app/interactions/locales/create.rb [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      create_place if external_place_id.present?
+      create_locale
+      add_locale_to_place if place
+      locale
+    end
+
+    def create_place
+      @place = compose(
```
Comment (ericcecchi) [inline app/interactions/locales/create.rb:21]: `compose` is a helper method for composing together interactions (i.e. interactions calling other interactions). It passes the given parameters to the given interaction and calls the `execute` method.

File: app/services/google/places/details.rb [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Services
+  module Google
+    module Places
+      # Fetches Place Details from the Places API
+      class Details < PlacesInteraction
```
Comment (ericcecchi) [inline app/services/google/places/details.rb:7]: Service objects are a pattern for abstracting calls to third party APIs (services). They are wrappers for the underlying libraries or API calls and can be helpful for triggering side effects or handling API errors in one place.

File: lib/tasks/deployment/20211006020953_seed_locales_places.rake [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+          name: 'Richmond, VA',
+          content: 'Historic capital city of Virginia',
+          hero_image_url: 'https://assets.simpleviewinc.com/simpleview/image/upload/c_fill,f_jpg,h_800,q_65,w_1920/v1/clients/virginia/CN19071201V_233_57a18292-4ab6-4633-b730-60cf1f0e531b.jpg'
+        },
+        external_place_id: 'ChIJ7cmZVwkRsYkRxTxC4m0-2L8'
+      }
+    ]
+
+    locales.each do |locale|
+      Locales::Create.run!(locale)
```
Comment (ericcecchi) [inline lib/tasks/deployment/20211006020953_seed_locales_places.rake:37]: This after party seeds locales by calling the `Locales::Create` interaction, which in turn creates places if the given `external_place_id` does not match an existing place.

**Review by ztcollazo — APPROVED** [review]

**Pull request merged**